### ISSUE TITLE
Reversed translation

### ISSUE THREAD

**Issue by NextTurn** [dotnet/docs.zh-cn#432]
https://docs.microsoft.com/zh-cn/dotnet/standard/base-types/regular-expression-language-quick-reference

https://github.com/dotnet/docs.zh-cn/blob/live/docs/standard/base-types/regular-expression-language-quick-reference.md

https://github.com/dotnet/docs.zh-cn/blob/a5269e6005f5b3f8771f39bf6574bca1fb1ae867/docs/standard/base-types/regular-expression-language-quick-reference.md#L84

The `\A` example translation is reversed.

**Comment by srvbpigh** [comment]
Hello, @NextTurn
 
Thank you for your feedback.
 
We are actively reviewing your comments and will get back to you soon.


Kind regards,
Microsoft DOCS International Team

**Comment by CeciAc** [comment]
Issue moved to [dotnet/docs #13979](https://github.com/dotnet/docs/issues/13979) via [**ZenHub**](https://www.zenhub.com/)

**Comment by CeciAc** [comment]
Hello, @NextTurn

Your contribution is much appreciated. Your feedback is related to the original documentation.
Hence, we are migrating this issue to the English repo. Please review the status of the issue here - https://github.com/dotnet/docs/issues/13979

We truly appreciate your contribution to our articles. 

Kind regards,
Microsoft DOCS International Team

**Comment by mairaw** [comment]
Reopened since this is not a source issue, but a localization issue.

**Comment by JasonCard** [comment]
@CeciAc I agree with @mairaw this is not source issue, but translation issue. For some reason, this translation was fixed a couple of times before, but it keeps getting reverted. Please help to look into why it was reverted again and fix the root cause.
cc @DavidPrio

**Comment by DavidPrio** [comment]
@JasonCard @mairaw we are working on this. Thanks!
@SunnyDeng @CeciAc cc
